sample-parent/sample-features/pom.xml

```diff
@@ -30,6 +30,7 @@
   <modules>
     <module>fixed-schema</module>
     <module>aftergroup-lastgroup</module>
+    <module>avaiableoutput</module>
```

Comment:
I think that we should use a more clear name for this component.
At least use kebab-case like others.
- output-flow
- output-definition